core/ui/PageControls/new-journal.tid

```diff
@@ -5,7 +5,7 @@ description: {{$:/language/Buttons/NewJournal/Hint}}
 
 \define journalButton()
 <$button tooltip={{$:/language/Buttons/NewJournal/Hint}} aria-label={{$:/language/Buttons/NewJournal/Caption}} class=<<tv-config-toolbar-class>>>
-<$action-sendmessage $message="tm-new-tiddler" title=<<now "$(journalTitleTemplate)$">> tags="$(journalTags)$" text="$(journalText)$"/>
+<$action-sendmessage $message="tm-new-tiddler" title=<<now """$(journalTitleTemplate)$ """>> tags=<<journalTags>> text=<<journalText>>/>
```

Comment:
```
"""$(journalTitleTemplate)$ """
 Extra Space? -------------^
```

Reply:
I knew, that it will be spotted :) 

That's by intention. It doesn't hurt, since saving a tiddler cut's it of and if it isn't there the title causes a problem if title has a trailing double-quote eg: `asdf"`

Reply:
If you have a better fix, just let me know. ... I wasn't really happy, but it was the simpliest fix I could find, that should have no side effects.